api/user.go

```diff
@@ -14,23 +14,22 @@ type UserUpdateParams struct {
 	Email    string                 `json:"email"`
 	Password string                 `json:"password"`
 	Data     map[string]interface{} `json:"data"`
+	AppData  map[string]interface{} `json:"app_metadata,omitempty"`
 }
 
 // UserGet returns a user
 func (a *API) UserGet(ctx context.Context, w http.ResponseWriter, r *http.Request) {
 	token := getToken(ctx)
 
 	user := &models.User{}
-	if result := a.db.First(user, "id = ?", token.Claims["id"]); result.Error != nil {
+	if result := a.db.Preload("AppMetaData").Preload("UserMetaData").First(user, "id = ?", token.Claims["id"]); result.Error != nil {
```

Comment:
instead of floating these strings everywhere maybe put it into a method like `userQuery(db *gorm.DB) *gorm.DB` and call it there